<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>conn_mysql_dialog</class>
 <widget class="QDialog" name="conn_mysql_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查询采集数据</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,9,2">
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>请选择查询方式</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按id查询</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按名称查询</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_find"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_find">
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>查询所有数据</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
